daskhub/Chart.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v2
+name: daskhub
+version: 0.0.1-set.by.chartpress
```

Comment:
Does chartpress set this from the tag?

Currently with the dask chart we increment this each release. It is semi-automated today with this script.

https://github.com/dask/helm-chart/blob/master/ci/release.sh#L9

We should probably bring them in line. If chartpress can do this automatically from git tags I'd be in favour of that.

Reply:
This came from https://github.com/pangeo-data/helm-chart/commit/66b7ccdca3016a9f1e46ab5dc38b7b85a22da31e (@consideRatio). But it looks like chartpress does indeed set it based on the tag + commit.

Reply:
Chartpress indeed sets the version based on a tag, number of commits since the tag, and the hash. But note that the tag needs to be on a commit in the branch, if the latest tag was on a tag not in the branch, chartpress would ignore it.

Reply:
Here's the output of `chartpress --long`


```diff
diff --git a/dask/Chart.yaml b/dask/Chart.yaml
index d79b1fd..ee832d7 100755
--- a/dask/Chart.yaml
+++ b/dask/Chart.yaml
@@ -1,7 +1,6 @@
----
 apiVersion: v1
 name: dask
-version: 4.3.1
+version: 4.3.1-n001.h7878309
 appVersion: 2.23.0
 description: Distributed computation in Python with task scheduling
 home: https://dask.org
diff --git a/daskhub/Chart.yaml b/daskhub/Chart.yaml
index 48e1b7d..ac7a094 100644
--- a/daskhub/Chart.yaml
+++ b/daskhub/Chart.yaml
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: daskhub
 icon: https://avatars3.githubusercontent.com/u/17131925?v=3&s=200
-version: 0.0.1-set.by.chartpress
+version: 4.3.1-n008.hfc9e4ce
 description: Multi-user JupyterHub and Dask deployment.
 dependencies:
   - name: jupyterhub
```

So things seems to be working OK. I assume that when we tag a release this will be set to the actual value?